packages/fxa-auth-server/scripts/scan-stripe-metadata.ts

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Stripe } from 'stripe';
+
+import { setupProcessingTaskObjects } from '../lib/payments/processing-tasks-setup';
+
+export async function init() {
+  const { stripeHelper } = await setupProcessingTaskObjects(
+    'scan-stripe-metadata'
+  );
+  const stripe: Stripe = stripeHelper.stripe;
+
+  const prices = [];
+  for await (const price of stripe.prices.list({
+    limit: 100,
+    product: 'prod_FvnsFHIfezy3ZI',
+    active: true,
+  })) {
+    prices.push({
+      name: price.nickname,
+      priceId: price.id,
+      interval: price.recurring?.interval,
+      productSet: price.metadata.productSet,
+      productOrder: price.metadata.productOrder,
+      // productSetOrder: price.metadata.productSetOrder,
+    });
+  }
+  console.table(prices);
+  return 0;
+}
+
+if (require.main === module) {
+  init()
+    .catch((err) => {
+      console.error(err);
+      process.exit(1);
+    })
+    .then((result) => process.exit(result));
```

Comment:
Note that `processing-tasks-setup` has a reference to statsd, and we have had issues losing statsd metrics because processes may exit before statsd has finsihed omitting everything. For an example of how to deal with this, see this: https://github.com/mozilla/fxa/blob/3469e6efabfe9c248e83e63e318eb41c9e6e927d/packages/fxa-admin-server/src/scripts/audit-tokens.ts#L484

Note that this might not even be applicable here, but I just wanted to call it out because it can be elusive to track down or even just detect.